pkg/services/authz/rbac/mapper.go

```diff
@@ -6,26 +6,63 @@ import (
 	"github.com/grafana/grafana/pkg/apimachinery/utils"
 )
 
+// Mapping maps a verb to a RBAC action and a resource name to a RBAC scope.
+type Mapping interface {
+	// action returns the action for the given verb.
+	// If no action is found, it returns false.
+	Action(verb string) (string, bool)
+	// scope returns the scope for the given resource name.
+	Scope(name string) string
+	// prefix returns the scope prefix for the translation.
+	Prefix() string
+	// AllActions returns all the actions for the translation.
+	AllActions() []string
```

Comment:
We need something like HasFolderSupport to avoid casting to `translation`
```suggestion
	AllActions() []string
	
	HasFolderSupport
```